string pos1
string vel1
string torq1

string pos2
string vel2
string torq2